### PR TITLE
WrapAround enabled for 19 Cell Cluster. Aliase turned to global in node struct

### PR DESCRIPTION
**Pull request by divyang-rawal**

### CODE REVIEW

File: deployment/deployment.go [hunk 1]
```diff
@@ -33,17 +33,18 @@ type Node struct {
...
+// func (n Node) Alias() int {
+// 	return n.Alias
+// }
 
-func (n *Node) SetAlias(a int) {
-	n.alias = a
+func (n Node) SetAlias(a int) {
+	n.Alias = a
+	// fmt.Println("updated alias, vid:", n.alias, a)
 }
```
Comment (wiless) [inline deployment/deployment.go]: why expose the Alias variable ? when there is function to access it

File: deployment/hexfunctions.go [hunk 2]
```diff
@@ -93,25 +82,34 @@ func Cube2XY(cube vlib.Location3D, hexsize float64) vlib.Location3D {
...
+			if prev != i {
+				vid = i
+
+			}
+
 		}
 	}
-	return distance, indx, dv
+	return vid
 }
```
Comment (wiless) [inline deployment/hexfunctions.go]: non backward compatible changes !! function arguments are different

**Review by wiless — CHANGES_REQUESTED** [review]
Please change based on the changes..
1. alias (private variable) to protect unintentional changes from outside, important information
2. ClosestMirror() is not backward compatible

File: deployment/deployment.go (hunk 1 above)
Comment (divyang-rawal) [reply deployment/deployment.go]: Changed it back to local. Wasn't working initially that's why turned it to global.

File: deployment/hexfunctions.go (hunk 2 above)
Comment (divyang-rawal) [reply deployment/hexfunctions.go]: Created new function for the same keeping the old as it was.

**Pull request merged**